app/ide-desktop/lib/content/src/index.ts

```diff
@@ -817,19 +775,23 @@ class Config {
     public email: string
     public application_config_url: string
     public test_workflow: string
+    public skip_min_version_check: boolean
+    public preferred_engine_version: SemVer
 
     static default() {
         let config = new Config()
         config.use_loader = true
         config.wasm_url = '/assets/ide.wasm'
         config.wasm_glue_url = '/assets/wasm_imports.js'
-        config.crash_report_host = cfg.defaultLogServerHost
+        config.crash_report_host = defaultLogServerHost
         config.data_gathering = true
         config.is_in_cloud = false
         config.entry = null
-        config.authentication_enabled = true
+        config.authentication_enabled = !Versions.isDevVersion()
```

Comment:
I remember someone was changing that in our team to be enabled NOT only in dev version. Ask pls people around.